testsuite/integration-arquillian/tests/pom.xml

```diff
@@ -1534,7 +1534,6 @@
                                     <auth.server.migration>true</auth.server.migration>
                                     <keycloak.migration.home>${containers.home}/auth-server-migration</keycloak.migration.home>
                                     <migration.import.props.previous>${migration.import.props.previous}</migration.import.props.previous>
-                                    <auth.server.feature>${auth.server.feature}</auth.server.feature>
```

Comment:
Yeah ... That was my first attempt and the reason why I forgot to remove it.

I don't think we want that because it would now force you to pass over this property whenever running migration tests and I think we should be opinionated about the features that should be enabled by default. It also avoids changes to the GHA. Much simpler and less error-prone, I would say ...

Reply:
Sorry, I misread your comment. If we add it to the `auth.server.feature` it will mess with other profiles. Or is it safe to do that? That is the reason why I not added the default value there.

Reply:
I'd assume it will be default soon anyway, so it shouldn't mess with other profiles.

Reply:
Are you sure? See https://github.com/keycloak/keycloak/blob/main/testsuite/integration-arquillian/tests/pom.xml#L528.

Reply:
Ok, you convinced me. Resolved!